src/views/Zniffer.vue

```diff
@@ -1025,6 +1042,68 @@ export default {
 				)
 			}
 		},
+		loadCapture() {
+			// Trigger file dialog
+			this.$refs.fileInput.click()
+		},
+		async handleFileSelect(event) {
+			const file = event.target.files[0]
+			if (!file) return
+
+			try {
+				await this.uploadAndLoadCapture(file)
+			} catch (error) {
+				this.showSnackbar(
+					`Error loading capture: ${error.message}`,
+					'error',
+				)
+			} finally {
+				// Reset file input
+				event.target.value = ''
+			}
+		},
+		uploadAndLoadCapture(file) {
+			return new Promise((resolve, reject) => {
+				const reader = new FileReader()
+
+				reader.onload = async (e) => {
+					try {
+						const byteArray = new Uint8Array(e.target.result)
+
+						const response = await this.sendAction(
+							{
+								apiName: 'loadCaptureFromBuffer',
+								buffer: Array.from(byteArray),
```

Comment:
Likewise, is this the way to upload a binary file?

Reply:
See comment above